changelog/Microsoft.DirectoryServices.json

```
@@ -66,7 +66,25 @@
       "WorkloadArea": "Identity and access",
       "SubArea": "Governance"
     },	  
-	{
+    {
+      "ChangeList": [
+        { 
+          "Id": "1c8b1db6-2c25-402f-9bcd-3f5f0fae5a53",
+          "ApiChange": "Method",
+          "ChangedApiName": "List transitiveRoleAssignments",
+          "ChangeType": "Addition",
+          "Description": "Added the [List transitiveRoleAssignments](https://docs.microsoft.com/en-us/graph/api/unifiedroleassignment-list-transitive) method.",
+          "Target": "unifiedRoleAssignment"
+        }
```

Please move your changelog to be the first object in the file, starting from line 3.
Added a changelist object for the new relationship
```suggestion
        {
          "Id": "1c8b1db6-2c25-402f-9bcd-3f5f0fae5a53",
          "ApiChange": "Relationship",
          "ChangedApiName": "transitiveRoleAssignments",
          "ChangeType": "Addition",
          "Description": "Added the **transitiveRoleAssignments** relationship to the [rbacApplication](https://docs.microsoft.com/en-us/graph/api/resources/rbacApplication?view=graph-rest-beta) resource.",
          "Target": "rbacApplication"
        },
        { 
          "Id": "1c8b1db6-2c25-402f-9bcd-3f5f0fae5a53",
          "ApiChange": "Method",
          "ChangedApiName": "transitiveRoleAssignments",
          "ChangeType": "Addition",
          "Description": "Added the [List transitiveRoleAssignments](https://docs.microsoft.com/en-us/graph/api/unifiedroleassignment-list-transitive) method to the [unifiedRoleAssignment](https://docs.microsoft.com/en-us/graph/api/resources/unifiedroleassignment?view=graph-rest-beta) resource.",
          "Target": "unifiedRoleAssignment"
        }
```

---

Moved to the top